extensions/linz/examples/collection.json

```diff
@@ -77,7 +77,9 @@
       "created": "2000-01-01T00:00:00Z",
       "updated": "2020-01-01T00:00:00Z",
       "file:checksum": "1220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
-      "proj:epsg": 32659
+      "proj:epsg": 32659,
+      "linz:encoding": "utf-8",
+      "linz:language": "en_NZ"
```

Comment:
[IETF language tags are defined to be separated by hyphens.](https://en.wikipedia.org/wiki/IETF_language_tag#Syntax_of_language_tags)